### PR TITLE
mon:Local variables("bufferlist proposal_bl") are not used

### PR DESCRIPTION
**Pull request by ShiqiCooperation**
Local variables("bufferlist proposal_bl") are not used

### CODE REVIEW

**Comment by tchaikov** [comment]
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#preparing-and-sending-patches

and put your Signed-off-by line at the end of the commit message?

**Comment by ShiqiCooperation** [comment]
title of commit message now is prefixed 。can you check again?

**Comment by tchaikov** [comment]
@ShiqiCooperation could you squash these two commit into a single one?

**Comment by ShiqiCooperation** [comment]
Why this commit failed?

**Comment by tchaikov** [comment]
retest this please.

File: src/mon/ConfigKeyService.cc [hunk 1]
```diff
@@ -58,8 +57,7 @@ void ConfigKeyService::store_put(string key, bufferlist &bl, Context *cb)
 }
 
 void ConfigKeyService::store_delete(string key, Context *cb)
-{
-  bufferlist proposal_bl;
+{
```
Comment (tchaikov) [inline src/mon/ConfigKeyService.cc]: please remove the trailing space

Comment (ShiqiCooperation) [inline src/mon/ConfigKeyService.cc]: trailing space is removed now , please check it .

**Comment by tchaikov** [comment]
could you squash these two commit into a single one?

**Comment by ShiqiCooperation** [comment]
OK , this two commit is squashed into a single one.

**Pull request merged**